query-node/schemas/council.graphql

```diff
@@ -46,6 +46,36 @@ enum ElectionProblem {
   NEW_COUNCIL_NOT_ELECTED
 }
 
+type CandidacyStatusActive @variant {
+  # no properties
+
+  # TODO: remove me - variant needs to have at least 1 property now
+  dummy: Int
+}
+
+type CandidacyStatusWithdrawn @variant {
+  # no properties
+
+  # TODO: remove me - variant needs to have at least 1 property now
+  dummy: Int
+}
+
+type CandidacyStatusElected @variant {
+  # no properties
+
+  # TODO: remove me - variant needs to have at least 1 property now
+  dummy: Int
+}
+
+type CandidacyStatusLost @variant {
+  # no properties
+
+  # TODO: remove me - variant needs to have at least 1 property now
+  dummy: Int
+}
+
+union CandidacyStatus = CandidacyStatusActive | CandidacyStatusWithdrawn | CandidacyStatusElected | CandidacyStatusLost
```

Comment:
Since all variants have no additional properties, perhaps using `enum` for `CandidacyStatus` would make things simpler.